public/js/main.js

```diff
@@ -21,7 +21,18 @@ $(document).on('ready', function() {
     }
   });
 
-  var slides = ['slide-dosomething', 'slide-counts', 'slide-campaigns', 'slide-reportbacks', 'slide-members', 'slide-ctl'];
+  var codes = {};
+  Papa.parse('/data/ISOCodes.csv', {
```

Comment:
Can you add a comment with what's going on here? I assume it has to do with the member map?